app/views/layouts/_navbar.haml

```diff
@@ -25,6 +25,7 @@
       .dropdown
         %button.btn.navbar-btn.dropbtn
           = current_user.name
+          &blacktriangledown;
```

Comment:
Something more like [this](http://getbootstrap.com/components/#dropdowns-example)?